### PR TITLE
Video Texture support for UVOL

### PR DESCRIPTION
**Pull request by CITIZENDOT**
## References
closes #_insert number here_

### CODE REVIEW

File: packages/engine/src/scene/constants/UVOLTypes.ts [hunk 1]
```diff
@@ -208,7 +208,7 @@ export interface GeometryInput {
   targets: Record<string, GLBTarget | DRACOTarget | UniformSolveTarget>
 }
 
-export type TextureFormat = 'ktx2' | 'astc/ktx2'
+export type TextureFormat = 'ktx2' | 'astc/ktx2' | 'vp9'
```
Comment (the-reality-engineer) [inline packages/engine/src/scene/constants/UVOLTypes.ts]: Does the fact that the video is encoded in `vp9` actually matter? Do we only support vp9? Couldn't we also support vp8 or h264 or h265 without any other changes to our code?

**Review by the-reality-engineer — CHANGES_REQUESTED** [review]

File: packages/engine/src/scene/constants/UVOLTypes.ts (hunk 1 above)
Comment (CITIZENDOT) [reply packages/engine/src/scene/constants/UVOLTypes.ts]: > Does the fact that the video is encoded in vp9 actually matter

It doesn't actually. Any video which is playable by `HTMLVideoElement` works.

Comment (the-reality-engineer) [reply packages/engine/src/scene/constants/UVOLTypes.ts]: Then we shouldn't use `vp9` as the texture format. The texture format should be `video`.

Comment (the-reality-engineer) [reply packages/engine/src/scene/constants/UVOLTypes.ts]: Looks like you did that already, thanks!

**Review by the-reality-engineer — APPROVED** [review]

**Pull request merged**